### PR TITLE
fix(lemnisk): add rudder message identifier flag

### PR DESCRIPTION
**Pull request by praveendsviz**
## PR Description

There are customers who have rudderstack based implementation of Lemnisk and native implementation of Lemnisk.  We are adding a flag to understand the rudderstack based calls.

## Notion ticket

Ticket link

## Screenshots

Not a UI change

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by shrouti1507 — CHANGES_REQUESTED** [review]
Need to update the test cases

File: src/integrations/Lemnisk/browser.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ class Lemnisk {
     // disabling eslint as message will be there iinn any case
     // eslint-disable-next-line no-unsafe-optional-chaining
     const { traits } = rudderElement.message?.context;
+    traits['isRudderEvents'] = 1;
```
Comment (koladilip) [inline src/integrations/Lemnisk/browser.js:76]: is it possible to use boolean value true instead of number 1

Comment (praveendsviz) [reply src/integrations/Lemnisk/browser.js]: This is done.

**Review by koladilip — APPROVED** [review]

**Comment by MoumitaM** [comment]
Hi @praveendsviz, Can you please sign the [CLA](https://rudderlabs.wufoo.com/forms/rudderlabs-contributor-license-agreement) so that we can go ahead with your changes?

**Pull request merged**